Fix #109 - Allow to handle custom hooks

micc83 opened the pull request:
A first implementation of what seen in #109. What do you think? I'm on the right lane?
Before going further (I still have to test update, list, delete commands) I'd like some feedback. Also not sure if `--skip-validation` is a good naming for what the option does. Maybe something like `--allow-custom` would be better?

BrainMaestro commented:
Hi @micc83 this looks good so far, and yes I agree that `--skip-validation` doesn't fit. Let's go with `--allow-custom-hooks`

micc83 commented:
All right, I've just renamed the option and added tests for the update, list and delete commands. One last concern is that right now custom hooks tests have duplicated initialization logic as some of the stuff made in the setUp and init methods is discarded. Another option could be to have a dedicated test class for custom hooks. Let me know what you think about it.

BrainMaestro commented:
@micc83 it's very good as is. thanks for the PR!
